network/tests/topology.rs

```diff
@@ -42,6 +42,42 @@ async fn test_nodes<F: Fn() -> TestSetup>(n: usize, setup: F) -> Vec<Node> {
     nodes
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn reconnect_nodes() {
+    // tracing_subscriber::fmt::init();
+    let setup = || TestSetup {
+        consensus_setup: None,
+        peer_sync_interval: 999999,
```

Comment:
I think the default is quite high, but no issue with it being bumped higher

Reply:
yeah, didn't want any peer syncs interfering with logic of the test.